UserScrape/crawler.py

```diff
@@ -392,7 +460,10 @@ def path_user(self)-> PurePath:
         return PurePosixPath(f'session_logs/{self.email}')
 
     def path_session(self) -> PurePath:
-        return PurePosixPath(f'session_logs/{self.email}/{self.init_time.strftime("%Y%m%d-%H%M%S")}.{self.driver.session_id}')
+        return PurePosixPath(f'session_logs/{self.email}/{self.init_time.strftime("%Y%m%d-%H%M%S")}.trial_{self.trial_nr}')
```

Comment:
I think the driver session does not give valuable information or does it? I replaced it with the trial number

Reply:
No, nothing valuable. This is better :)